sklearn/model_selection/tests/test_search.py

```diff
@@ -210,7 +210,7 @@ def check_hyperparameter_searcher_with_fit_params(klass, **klass_kwargs):
                          "Expected fit parameter(s) ['eggs'] not seen.",
                          searcher.fit, X, y, spam=np.ones(10))
     assert_raise_message(AssertionError,
-                         "Fit parameter spam has length 1; expected 4.",
+                         "Fit parameter spam has length 1; expected",
```

Comment:
what's happening here?

Reply:
The fit_params had been indexed according to the cv